gong-notifier-github-status/README.md

```diff
@@ -61,29 +60,32 @@ There are two ways to configure the access token:
 1. As a secret variable `GONG_STATUS_AUTH_TOKEN`. The variable must be defined on the pipeline, not on a stage or job.
```

Comment:
The 2nd way of configuring the plugin suggests that if the material is configured with credentials, those credentials will be reused for the feedback. For private repositories, this seems the way to go. Do I understand correctly that the explicit `GONG_STATUS_AUTH_TOKEN` setting is for GitHub repositories that can be accessed anonymously?

Also, based on the above, is it true that just by installing the plugin one can get GitHub build feedback for private repositories with no additional configuration?

Reply:
> Do I understand correctly that the explicit GONG_STATUS_AUTH_TOKEN setting is for GitHub repositories that can be accessed anonymously?

Correct, or for any other reason you might want a different identity to do the status updates.

> Also, based on the above, is it true that just by installing the plugin one can get GitHub build feedback for private repositories with no additional configuration?

Yep that's the idea.